docs/_data/roadmap.yml

```diff
@@ -0,0 +1,33 @@
+- 
+  term: "Active development"
+  items: 
+    -
+      name: Unify Camera API into a single method
+      issues: ["3583", "2801"]
+    -
+      name: Augmenting "data" events
+      issues: ["3577", "3578"]
+    - 
+      name: Vertical text rendering improvements
+      issues: ["3438", "3470", "3402"]
+-
+  term: "Upcoming"
+  items:
+    -
+      name: Improve performance around setting styles with <code>setStyle()</code>
+      issues: ["1370", "1989"]
+    -
+      name: Fix GeoJSON memory leak
+      issues: ["3220", "3595"]
+    -
+      name: Custom source type API
+      project: 2
+-
+  term: "Long-term"
```

Comment:
I think it's a great idea to just have two columns to start with. We can add a third if it feels necessary during Tuesday scrum.